docs/customization.rst

```diff
@@ -104,12 +104,17 @@
 Customization
 =============
 
-This theme is customized via fields in the `html_theme_options` `dict` in *conf.py*.
-Unlike, newer versions of mkdocs-material theme, this theme also supports the use of a textual
-"hero" section.
+Metadata for a single page
+==========================
+
+Each page can support a set of page-specific options. These are configured using metadata roles.
```

Comment:
Add link to sphinx documentation:

https://www.sphinx-doc.org/en/master/usage/restructuredtext/field-lists.html#file-wide-metadata

Could probably use intersphinx as well rather than hard-coding the URL.

Should be called "metadata fields" rather than roles.

Reply:
we are using intersphinx for the sphinx-doc master docs. I had to change a link about `html_logo` because it was defaulting to this theme's <code>:confval:\`html_logo\`</code>

Reply:
I'm not seeing a way to use intersphinx for that link (its not a `confval` or of a domain).

Reply:
I did find a way to force the `html_logo` link to default to the sphinx docs using intersphinx 🤓